src/app.js

```diff
@@ -714,6 +715,8 @@ var run = function () {
     })
   })
 
+  $('#createAccount').click(function () { udapp.newAccount() })
```

Comment:
The UI must be refreshed (to populate the account dropdown) after a new account is created.